pkg/registry/apis/provisioning/webhook.go

```diff
@@ -99,7 +99,7 @@ func (s *webhookConnector) Connect(ctx context.Context, name string, opts runtim
 			job := &provisioning.Job{
 				ObjectMeta: v1.ObjectMeta{
 					Namespace: namespace,
-					Name:      fmt.Sprintf("%s:%s", repo.Config().GetName(), rsp.Job.Action),
+					Name:      fmt.Sprintf("%s-%s", repo.Config().GetName(), rsp.Job.Action),
```

Comment:
will the naming convention work here?  Unlike sync/migrate where we only want one at a time -- every PR request should be processed (even if other jobs are queued)